templates/errors/error.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'main/base.html' %}
+{% block title %} {{ code }} - Internal server error {% endblock %}
+{% block beta_error %}{% endblock %}
+{% block content %}
+    <script>
+        window._RequestMethod = "{{ request.method.lower() }}";
+        window._Code = {{ code }};
+        window._ErrorMsg = "{{ error_message }}"
+        window._Path = "{{ request.path }}"
+    </script>
+    <div class="uk-background-muted">
+        <div class="uk-section">
+            <div class="uk-text-center"><code class="uk-text-large">{{ error_title }}</code></div>
+        </div>
+    </div>
+    <script src="/static/js/typewriter.js"></script>
+    <link href="/static/css/window.css" rel="stylesheet" type="text/css"/>
+    <div class="uk-container uk-section">
+        <div class="window">
+            <div class="top">
+                <div class="panel">
+                    <span class="first"></span>
+                    <span class="second"></span>
+                    <span class="third"></span>
+                </div>
+
+                <div class="nav">
+                    <span class="prev active"><i class="fa fa-angle-left"></i></span>
+                    <span class="next"><i class="fa fa-angle-right"></i></span>
+
+
+                </div>
+
+
+            </div>
+
+            <div class="inside">
+                <div class="blok">
+                    <pre id="terminal"><code style="white-space:pre-wrap;" class="python" id="error"></code></pre>
+                </div>
+            </div>
+
+        </div>
+    </div>
+
+    <style>
+        #terminal {
+            background-color: black;
+            color: white;
+        }
+    </style>
+
+    <noscript>
+        <h1>There was an issue processing your request</h1>
+        <p>Please try again in a little while, if the problem persists, feel free to open a bug on our <a
```

Comment:
"Try again after a while. If the problem persists, open a bug on our __GitHub issue tracker.__"